extensions-contrib/opentelemetry-extensions/src/main/java/org/apache/druid/data/input/opentelemetry/protobuf/OpenTelemetryMetricsProtobufReader.java

```diff
@@ -145,14 +145,16 @@ private List<InputRow> parseMetric(Metric metric, Map<String, Object> resourceAt
       case SUM: {
         inputRows = new ArrayList<>(metric.getSum().getDataPointsCount());
         metric.getSum()
-            .getDataPointsList()
+            // https://github.com/open-telemetry/opentelemetry-collector/blob/2ebe1e501f9a206237de1009a127f1a419ab3696/pdata/internal/data/protogen/metrics/v1/metrics.pb.go#L123-L139
+            .getDataPointsList().stream().filter(x -> (x.getFlags() & 1) != 1)
```

Comment:
Create a constant like `DATA_POINT_FLAGS_NO_RECORDED_VALUE` for the magic value 1